src/gdal/rasterband.jl

```diff
@@ -257,8 +257,8 @@ _getrasterscale(hBand::GDALRasterBandH, pbSuccess::Ptr{Cint}) =
     GDALGetRasterScale(hBand, pbSuccess)::Cdouble
 
 function getrasterscale(rasterband::GDALRasterBandH)
-    success = Ref{Cint}()
```

Comment:
I see. It fails because the function signature is `pbSuccess::Ptr{Cint}` and
```julia
Ref{Cint} <: Ptr{Cint} # --> false
Ptr{Cint} <: Ref{Cint} # --> true
```

I believe rather than allocating array objects it would be better to change to function signature to `Ref{Cint}`, or remove it entirely as done in [GDAL.jl](https://github.com/visr/GDAL.jl/blob/8f8d3f22072615fd8c1eb4f8954241eede3bfc25/src/gdal_h.jl#L1836)